packages/api-bindings/package.json

```diff
@@ -69,11 +69,18 @@
     "jest": "^29.3.1",
     "rimraf": "^3.0.2",
     "ts-jest": "^29.0.3",
-    "typescript": "^4.9.3",
-    "tsup": "^6.5.0"
+    "typescript": "^4.9.3"
   },
   "peerDependencies": {
-    "react": "^18.2.0"
+    "react": "^18.2.0",
+    "@faker-js/faker": "^7.6.0"
```

Comment:
Not 100% sure if we need this here, potentially a workaround but `preconstruct` was upset with us using `mocks` as an entry point and exporting things that may use `@faker-js/faker`

Reply:
Can you please clarify "upset"? In what sense and why adding the fakerjs as peer dep solves it?

Reply:
Preconstruct requires that all packages that are imported by a package be specified in dependencies or peerDependencies. This is to avoid the mistake of publishing a package that imports a package that isn't specified in its dependencies causing a consumer to be unable to use the package.

Reply:
that is really no go IMO as would throw the 
```
`@faker-js/faker` is missing but required by `@lens-protocol/api-bindings` 
```
on the consumer side when installing `@lens-protocol/react`.

Will have a look on monday at how we can solve that in a way that would not affect consumers.

Reply:
Same affects all packages that have modified `peerDependencies`

Reply:
Agreed, that would be less than ideal.

Will explore our options around this

Reply:
I guess moving `faker` to dependencies (not devDeps) would also solve it, or we don't export mocks at all.
`preconstruct ` is correct that if we want to export `mocks` and they use `faker`, then `faker` has to be part of the bundle or a peer dep, no?